xml/schema/Core/Job.xml

```diff
@@ -61,6 +61,13 @@
     <add>4.1</add>
   </field>
   <field>
+    <name>scheduled_run_date</name>
+    <type>timestamp</type>
+    <default>NULL</default>
```

Comment:
Try removing this - I think required=FALSE (or omitting required) is equivalent to default is NULL - but I'm not sure it's valid to explicitly set default to NULL